src/plays/photoshopApp/SidebarItem.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+export default function SidebarItem({ name, active, handleClick }) {
+  return (
+    <button className={`sidebar-item ${active ? 'active' : ''}`} onClick={handleClick}>
+      {name}
+    </button>
+  );
+}
```

Comment:
I would recommend using the arrow function to improve the readability.

Reply:
Done and Committed Sir